snapshots/ophi/2022-12-13/multidimensional_poverty_index.py

```diff
@@ -0,0 +1,37 @@
+"""Ingest OPHI raw data: Multidimensional POverty Index.
+The file is not yet available in the website, so for now the data is uploaded manually
```

Comment:
It would be convenient to explain in detail what needs to be done to get that file manually, e.g. "Go to this website, click on this, etc."